### DISCUSSION TITLE
Stability Matrix UI crashed - only SD-UI working.

### DISCUSSION THREAD

**Discussion by zentsang** [LykosAI/StabilityMatrix#76] [Q&A] [answered]
Running the latest Stability Matrix with all the default settings during setup. Ran a few tests and then it crashed. However, after install there it no sign of a Shortcut, BAT, or EXE to open it again. I can run the stable-diffusion-webui just fine with the webui.bat file. I just have no idea where the main start file is for the Stability Matrix interface.

**Comment by ionite34** [comment, marked as answer]
Did you delete the downloaded app executable? There is no "installer" for stability matrix essentially, the app is self contained in the file. 

You can download / unpack the executable again to run, and it should keep the existing settings.

**Reply by zentsang** [reply]
DOH! I just didn't think the downloaded executable was both the setup and the main UI all-in-1. I didn't delete and just ran it. All is well. Thanks for responding. Maybe my noted user-error will help others in the future.

**Reply by ionite34** [reply]
Np, glad it worked. You can also use the "Add Stability Matrix to the Start Menu" button in settings to be able to find it in Windows search / start menu like normal "installed" apps.
